src/fuzz_introspector/frontends/frontend_cpp.py

```diff
@@ -563,6 +569,10 @@ def dump_module_logic(self,
                           harness_source: str = '',
                           dump_output=True):
         """Dumps the data for the module in full."""
+        _ = entry_function
+        _ = harness_name
```

Comment:
can we do some `# pylint disable` stuff instead of this?

Reply:
yes, changed.